docs/components/navigation-primary.njk

```diff
@@ -35,251 +31,498 @@ tags:
 
-  <div class="inline-stacked">
-    <img src="{{ '/assets/navigation/navigation-usage-1.svg' | url }}" alt="Navigation layout" style="">
+  <img src="{{ '/assets/navigation-primary/primary-nav-usage-intro.png' | url }}" alt="Primary navigation - usage" style="--inline-img-max-width: 1000px;">
 
-    <img src="{{ '/assets/navigation/navigation-usage-2.svg' | url }}" alt="Navigation layout" style="">
+  <h4>Full-width expandable tray</h4>
 
-    <img src="{{ '/assets/navigation/navigation-usage-3.svg' | url }}" alt="Navigation layout" style="">
+  <p>Menus will trigger an expandable tray when selected and include content or tasks that are specific to the website in which it is used. Use the expandable tray to organize a large amount of content in two, three, or four columns.</p>
+
+  <div class="margin-bottom--3">
+    <p style="background-color: #fdf7e7; border-top: 2px solid #f0ab00; padding: 16px 16px 16px 24px; font-size: 14px; letter-spacing: 0.2px;"><span style="color:#795600"><strong>Warning</strong></span><br />Do not use more than 4 columns in a full-width expandable tray, consider using a component like <a href="../tabs">Tabs</a> instead for more organization options.</p>
   </div>
 
-  <h4>Positioning</h4>
+  <img src="{{ '/assets/navigation-primary/full-width-tray-4-columns.png' | url }}" alt="Primary navigation - full-width expandable tray (four columns)" style="--inline-img-max-width: 1000px;">
 
-  <p>Navigation is always persistent and is located at the very top of the page on all screens. In certain situations, other components can be placed above the navigation, like an <a href="../announcement">Announcement banner</a>.</p>
+  <p style="padding-top: 24px;">If content is organized in less than four columns, containers will stretch to fill the columns.</p>
 
-  <div class="inline-stacked">
+  <div class="margin-bottom--5">
+    <img src="{{ '/assets/navigation-primary/full-width-tray-3-columns.png' | url }}" alt="Primary navigation - full-width expandable tray (three columns)" style="--inline-img-max-width: 1000px;">
+  </div>
 
-    <img src="{{ '/assets/navigation/navigation-positioning-1.svg' | url }}" alt="Navigation positioning" style="">
+  <img src="{{ '/assets/navigation-primary/full-width-tray-2-columns.png' | url }}" alt="Primary navigation - full-width expandable tray (two columns)" style="--inline-img-max-width: 1000px;">
 
-    <img src="{{ '/assets/navigation/navigation-positioning-2.svg' | url }}" alt="Navigation positioning with banner" style="">
+  <h4>Fixed-width expandable tray</h4>
 
+  <p>Sometimes a small amount of content can be placed in a fixed-width expandable tray and organized in one or two columns. The size of the expandable tray depends on the amount of content, however, utilities are not able to leverage fixed-width expandable trays at this time.</p>
```

Comment:
I think the second sentence (`The size of...`) could be broken up into two sentences for clarity.